File: _TestSDK/_TestSDK/_TestSDK.m
//
//  _TestSDK.m
//  _TestSDK
//
//  Created by 김기현 on 2020/05/25.
//  Copyright © 2020 wimes. All rights reserved.
//

#import "_TestSDK.h"

//int main (int argc, const char* argv[]){
//    @autoreleasepool {
//        TestSDK* testSDK = [[TestSDK alloc] init];
//    }
//
//    return 0;
//}

@implementation _TestSDK
- (id)init
{
    self = [super init];
    if (self) {
        testSDK = [[TestSDK alloc] init];
    }
    return self;
}

- (void) login{
    [testSDK login];
}
- (void) purchase{
    [testSDK purchase];
}

@end

File: TestSDK/TestSDK/TestSDK.swift
//
//  TestSDK.swift
//  TestSDK
//
//  Created by 김기현 on 2020/05/25.
//  Copyright © 2020 wimes. All rights reserved.
//

import Foundation

#if canImport(Member)
    import Member
#endif
#if canImport(Billing)
    import Billing
#endif

@objc public class TestSDK: NSObject{
    
    @objc public func login(){
        print("TestSDK.login()")
        
        #if canImport(Member)
        let member = Member()
        member.login()
        #else
        print("Do not find module Member")
        #endif
        
    }
    
    @objc public func purchase(){
        print("TestSDK.purchase()")
        
        #if canImport(Billing)
        let billing = Billing()
        billing.purchase()
        #else
        print("Do not find module Billing")
        #endif
        
    }
}
